DS-And-Algo/index.html

```diff
@@ -153,6 +155,16 @@ <h4>
                     </h4>
                 </a>
             </li>
+             
+
+            <li>
+                <a href="./answers/shubhamr238-q4.html">
+                    <h4>
+                       Puppy Treat (Array)
```

Comment:
Explain question bit here as well.

Reply:
this was a mock interview question

Reply:
it is difficult to explain the question there. it's just an array question.

Reply:
i have explained the whole question in the html file

Reply:
Yeah but still write a bit about the question there as well